### DISCUSSION TITLE
Dead link

### DISCUSSION THREAD

**Discussion by ocmca** [twbs/bootstrap#37865] [General] [answered]
Sorry if this is a repost, but I get a 404 when clicking... v.1.10.3

![image](https://user-images.githubusercontent.com/10720140/212084505-afe364e9-64d1-48c2-be63-2a66b493735d.png)

at https://icons.getbootstrap.com/

**Comment by julien-deramond** [comment, marked as answer]
Thank you for reporting it @ocmca. A corresponding issue exists here: https://github.com/twbs/icons/issues/1555. We'll try to tackle it asap :)
